.github/workflows/release.yml

```diff
@@ -0,0 +1,131 @@
+name: release
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+env:
+  PACKAGE_DIR: adbdpyg_adapter
+  TESTS_DIR: tests
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: ["3.7", "3.8", "3.9"]
```

Comment:
```suggestion
        python: ["3.7", "3.8", "3.9", "3.10"]
```

Reply:
Unfortunately we aren't yet able to introduce `3.10` support, since the [torch](https://pypi.org/project/torch/) dependency does not yet support `3.10`

Reply:
According to the PyG docs 3.10 is supported - probably worth testing it? If I'm not mistaken PyTorch 1.11 also introduced wheels for 3.10 :)

Reply:
ah good to know! running a 3.10 build now

Reply:
good catch! `3.10` support has been added as of https://github.com/arangoml/pyg-adapter/pull/2/commits/2158581d3daa4e59ff1e5c87499deb5040d64200